api/src/database/run-ast/run-ast.ts

```diff
@@ -89,7 +89,18 @@ export async function runAst(
 
 		// Run the items through the special transforms
 		const payloadService = new PayloadService(collection, { knex, schema });
-		let items: null | Item | Item[] = await payloadService.processValues('read', rawItems, query.alias ?? {});
+
+		// Include aggegation e.g. "count->id" in alias map
+		const aggregateMapped = Object.fromEntries(
```

Comment:
As discussed it may be worthwhile to pass down the aggregates to `processValues` and process the information there accordingly (e.g. in `processDates`). This will ensure all "processValues` has all our processing logic.

P.S. We can also update the `processAggregates` to utilize this new property to ensure no conflicts with any alias names that contain `->`

Reply:
Did a clean up, let me know if you have more suggestions!